addons/l10n_ua/data/account_tax_template.xml

```diff
@@ -0,0 +1,485 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data noupdate="0">
+        <record id="ua_psbp_6412" model="account.account.template">
+            <field name="tag_ids" eval="[(6,0,[ref('l10n_ua.acc_tag_vat')])]"/>
+        </record>
+
+        <!-- Tax template for VAT -->
+        <record id="sale_tax_template_vat20_psbo" model="account.tax.template">
+            <field name="chart_template_id" ref="l10n_ua_psbo_chart_template"/>
+            <field name="sequence">10</field>
+            <field name="name">Реалізація ПДВ 20%</field>
+            <field name="description">+ ПДВ 20%</field>
+            <field name="amount">20</field>
+            <field name="type_tax_use">sale</field>
+            <field name="account_id" ref="ua_psbp_6431"/>
+            <field name="refund_account_id" ref="ua_psbp_6431"/>
+            <field name="tag_ids" eval="[(6,0,[ref('l10n_ua.tax_tag_vat20')])]"/>
+            <field name="tax_group_id" ref="l10n_ua.tax_group_vat20"/>
+        </record>
+        <record id="sale_tax_template_vat20incl_psbo" model="account.tax.template">
+            <field name="chart_template_id" ref="l10n_ua_psbo_chart_template"/>
+            <field name="sequence">10</field>
+            <field name="name">Реалізація в т. ч. ПДВ 20%</field>
+            <field name="description">в т. ч. ПДВ 20%</field>
+            <field name="amount">20</field>
+            <field name="type_tax_use">sale</field>
+            <field name="price_include" eval="1"/>
+            <field name="account_id" ref="ua_psbp_6431"/>
+            <field name="refund_account_id" ref="ua_psbp_6431"/>
+            <field name="tag_ids" eval="[(6,0,[ref('l10n_ua.tax_tag_vat20')])]"/>
+            <field name="tax_group_id" ref="l10n_ua.tax_group_vat20"/>
+        </record>
```

Comment:
accounting in Ukraine, and probably for all CIS countries, more complex, for instance in  invoicing we have two documents invoice and act, they like proforma invoice and invoice,  but invoice and act are not same and they both need to exist in system, they have different dates, and reconciliation could be provide on each of it,
 other case is for stock picking, we have stock picking to customer and tax picking and they could be different, and tax picking should be registered in tax department (each picking should be registered in tax department during 15 days!), and on stock picking we make move of products
so this module in generally will not provide localization in whole scope, but it would be easier to work with, if at least would have same chart

Reply:
Well, we have law changes since 2017. Now paid invoice is the same as "act of accomplished work". Proforma and invoice works fine. We have solved most of the problems during implementations for customers with customization  of invoicing process. 
But in general proforma and invoice may work out of the box.

Same for stock (outgoing picking). If it is implemented by "1C" in some way - it doesn't mean that it is only one way to do things.

The idea for vat tags is to have ability to distinguish vat taxes from other one's when they are created manually. For example if company has excise taxes we need to create vat taxes with child taxes.

Reply:
What amounts need to be filed for the vat report?  (total purchase/sale net amount / tax amount /...)  Because normally we adapt tags to that.  I could not find the vat report easily on the internet.